src/server/actions/contest/addContest.ts

```diff
@@ -19,7 +19,7 @@ const addContestRequestSchema = z.object({
 
 export type AddContestRequest = z.infer<typeof addContestRequestSchema>
 
-const addContestAction = async (request: AddContestRequest) => {
+export const addContestAction = async (request: AddContestRequest) => {
```

Comment:
Decided to export the unserialized version of the server actions for better type resolution and assertion when testing.